install/kubernetes/cilium/charts/operator/templates/deployment.yaml

```diff
@@ -173,6 +173,9 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
 {{- end }}
       restartPolicy: Always
+{{- if and (eq .Release.Namespace "kube-system") (or (gt .Capabilities.KubeVersion.Minor "10") (gt .Capabilities.KubeVersion.Major "1"))}}
+      priorityClassName: cluster-essential
```

Comment:
I assume this should be `priorityClassName: system-cluster-critical`?

Reply:
I changed it, I didn't realize `cluster-essential` was a custom priorityClass in our cluster.